.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Release workflow
+jobs:
+  build-phar:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: 7.4
+          ini-values: memory_limit=2G, display_errors=On, error_reporting=-1, phar.readonly=0
+          tools: pecl, phive
```

Comment:
why pecl ?

Reply:
Mostly copy past from what I used to test, removed it.